lib/ansible/galaxy/data/default/collection/meta/runtime.yml

```diff
@@ -0,0 +1,2 @@
+---
+requires_ansible: '>=2.9.10'
```

Comment:
This file isn't required to use a collection and collections can be used before 2.9.10, so I'm wondering if this limitation should be added to the default skeleton. #74901 is related, it adds a new skeleton with this file to make publishing community collections easier.

Reply:
we could lower the version number. The pain I'm trying to address is you can't upload the collection to galaxy without this file

Reply:
Yeah. I'm  saying `ansible-galaxy collection init` is used for more than publishing collections. No need to change anything unless others weigh in though, it's just my opinion.

Reply:
Making it a comment works for me :+1: